data/releases/eol.yaml

```diff
@@ -50,10 +50,6 @@ branches:
   finalPatchRelease: 1.18.20
   note: Created to solve regression introduced in 1.18.19
```

Comment:
Aside: this is not a note about v1.18; there's some challenge here about how we explain the available notes to readers.

Another PR can fix that, I hope.